app/src/main/java/com/locoquest/app/MainActivity.kt

```diff
@@ -136,4 +145,30 @@ class MainActivity : AppCompatActivity(), OnMapReadyCallback{
     override fun onMapReady(p0: GoogleMap) {
 
     }
+}
+fun main() {
+    val retrofit = Retrofit.Builder()
+        .baseUrl("https://geodesy.noaa.gov/api/nde/")
+        .addConverterFactory(MoshiConverterFactory.create())
+        .build()
+
+    val benchmarkDAO = retrofit.create(IBenchmarkDAO::class.java)
+
+    val pid = "AB1234"
+
+    val call = benchmarkDAO.getBenchmarkByPid(pid)
+    call.enqueue(object : Callback<Benchmark> {
+        override fun onResponse(call: Call<Benchmark>, response: Response<Benchmark>) {
+            if (response.isSuccessful) {
+                val benchmark = response.body()
+                println(benchmark)
+            } else {
+                println("Error: ${response.code()}")
+            }
+        }
+
+        override fun onFailure(call: Call<Benchmark>, t: Throwable) {
+            println("Error: ${t.message}")
+        }
+    })
```

Comment:
This is a DAO operation, which belongs in the DAO layer... definitely not in the Activity.